ingest_api/runtime/requirements.txt

```diff
@@ -8,7 +8,7 @@ orjson>=3.6.8
 psycopg[binary,pool]>=3.0.15
 pydantic_ssm_settings>=0.2.0
 pydantic>=1.10.12
-pypgstac==0.7.4
+pyjwt>=2.8.0
```

Comment:
this came from merging upstream develop changes. The same line and same first two characters confused the merge conflict resolver

Reply:
thanks for pointing this out! I got rid of `pyjwt`.. because looks like it was no longer in the `develop` branch and added `pypgstac` back

Reply:
Yeah I guess we get it here now https://github.com/NASA-IMPACT/veda-backend/blob/develop/common/auth/setup.py#L6

Reply:
But we still do need `- pypgstac==0.7.4`